frontend/src/components/smart/PvPArenaInfo.vue

```diff
@@ -1,53 +1,101 @@
 <template>
   <div>
-    <h1 class="title">ARENA INFORMATION</h1>
+    <h1 class="title">
+      {{$t('pvp.arenaInformationCaps')}}
```

Comment:
Definietly don't ever use CAPS in translations, just use text-uppercase css class